src/test/resources/testConfig.properties

```diff
@@ -1,2 +1,3 @@
 github.oauth.token=351wa351d38aw4c97w98f7987ew98f987we97gs4
-template.format.url=https://www.someniceurlhere.org/
\ No newline at end of file
+template.format.url=https://www.someniceurlhere.org/
+whitelist.enabled=true
```

Comment:
tests that haven't been accounting for whitelisting shouldn't be affected (in other words you don't need whitelisting for all tests as it may be a performance bottleneck)